requirements.txt

```diff
@@ -1 +1 @@
-ansible
\ No newline at end of file
+ansible==2.2.1.0
```

Comment:
You're right, I've just run into it with another role. I think this is the bug: https://github.com/ansible/ansible/issues/23016